clang/lib/Basic/Targets/OSTargets.cpp

```diff
@@ -259,6 +259,7 @@ static void addVisualCDefines(const LangOptions &Opts, MacroBuilder &Builder) {
 
 void addWindowsDefines(const llvm::Triple &Triple, const LangOptions &Opts,
                        MacroBuilder &Builder) {
+  Builder.defineMacro("__UEFI__");
```

Comment:
What's the rationale for doing this for Windows targets? It seems to me like this would make the define significantly less helpful?

Reply:
Removed it from Windows target.